pennylane/optimize/adam.py

```diff
@@ -61,7 +62,7 @@ def __init__(self, stepsize=0.01, beta1=0.9, beta2=0.99, eps=1e-8):
         self.sm = None
         self.t = 0
 
-    def apply_grad(self, grad, x):
+    def apply_grad(self, grad, args):
```

Comment:
Don't forget to update the docstring below!

Reply:
and looks like that one is fixed too.